### PR TITLE
Introduce new package structure

### LINKED ISSUE #344: Introduce package structure for server
The server codebase grows and needs a better package structure.

### PR DESCRIPTION
**Pull request by fmueller**
Closes #344

### CODE REVIEW

File: server/src/main/java/de/zalando/zally/external/jbossdna/Inflector.java [hunk 1]
```diff
@@ -267,20 +238,18 @@ public String upperCamelCase( String lowerCaseAndUnderscoredWord,
...
+     * @param delimiterChars              optional characters that are used to delimit word boundaries
      * @return the camel case version of the word
      * @see #underscore(String, char[])
      * @see #upperCamelCase(String, char[])
      * @see #lowerCamelCase(String, char[])
      */
-    public String camelCase( String lowerCaseAndUnderscoredWord,
-                             boolean uppercaseFirstLetter,
-                             char... delimiterChars ) {
+    public String camelCase(String lowerCaseAndUnderscoredWord,
```
Comment (fmueller) [inline server/src/main/java/de/zalando/zally/external/jbossdna/Inflector.java]: False Positive - this is copied code from another library.

**Comment by netme** [comment]
I've checked Travis logs, seems to be the error is caused by merging #340 . I will add an extra commit which will fix that.

**Comment by netme** [comment]
👍

**Comment by netme** [comment]
👍

**Comment by netme** [comment]
@maschleg , @maxim-tschumak could you please also check it?

**Pull request merged**